Thinktecture.Relay/Server/AcknowledgmentMode.cs

```diff
@@ -0,0 +1,24 @@
+namespace Thinktecture.Relay.Server
+{
+	public enum AcknowledgmentMode
+	{
+		/// <summary>
+		/// Requests in the message queue will be automatically acknowledged by
```

Comment:
`Requests in the message queue will be automatically acknowledged by the on-premise before contacting the local target`